adapter/internal/oasparser/envoyconf/routes_with_clusters.go

```diff
@@ -810,7 +812,7 @@ func genRouteCreateParams(swagger *model.MgwSwagger, resource *model.Resource, v
 		endpointBasePath:  endpointBasePath,
 		corsPolicy:        swagger.GetCorsConfig(),
 		resourcePathParam: "",
-		resourceMethods:   nil,
+		resourceMethods:   []string{"GET"},
```

Comment:
Will this add default get resource to REST APIs as well

Reply:
For REST , there's a check to see whether resources are nil or not. 
https://github.com/wso2/product-microgateway/blob/6784b0c44b57aaaca842c1247398f0d2f50e7d87/adapter/internal/oasparser/envoyconf/routes_with_clusters.go#L818

For REST APIs if no resources, then the default will be `GET`. Is that an issue ?

Reply:
fixed by adding a method to return resourceMethods